whitelist.full.txt

```diff
@@ -29,7 +29,8 @@ login.live.com>*
 
 # StreamComplet.com video playback & spurious redirects (#40)
 #  - Last checked: 17.04.2017
```

Comment:
Please update the domain name in the comment and the "last checked" date as well